### PR TITLE
Feature/wine with food dropdown

### PR DESCRIPTION
**Pull request by timweyel**
-Added function to remove existing elements
-Added function to add wineCardGenerator
-When wine dropdown option is selected the above actions occur

### CODE REVIEW

File: assets/images/grape-logo.svg [hunk 1]
```diff
@@ -0,0 +1 @@
..."M84.86,121.65l15.32-4.7c.4,16.51-3.75,29.34-17.92,37.72-8.57,5.08-11.37,16.29-7.18,24.93,3.83,7.92,13.93,12.75,22.16,10.58,10-2.61,15.86-11.36,14.73-21.84-.14-1.28-.26-2.58-.37-3.79h13.83c3.88,8.61-1,26.37-13.68,34.47-11.57,7.39-24,8-35.92,1.22-11.42-6.51-17.15-17-17-30.1.14-10.85,4.79-20,13.93-26.12C80.66,138.71,86.91,132.8,84.86,121.65Z" transform="translate(207.26 95.83)"/><path class="cls-2" d="M96.33,151.19l7.72-13.11c15.34,8.52,26.49,7.31,32.43-3.53,6.24-11.4,1.73-21.3-13.48-29.44l7.66-13.06c21.28,11.17,26.51,28.51,19.9,45.7C146.44,148.44,138.23,155.28,127,158,116,160.59,106.13,156.71,96.33,151.19Z" transform="translate(207.26 95.83)"/><path class="cls-2" d="M54.59,91.72c2.69,4.58,5.17,8.81,7.95,13.53-8.94,3.49-15.83,8.59-15.86,19.49s6.55,16.3,16,20c-2.51,4.07-4.82,7.79-7.1,11.48-8.44-.25-20-11-22.25-22.72C30.56,119,31.5,105.9,54.59,91.72Z" transform="translate(207.26 95.83)"/><path class="cls-3" d="M87.2,40.25c-2.26-13.35-11.1-19.06-23-21.48V6.08c19.35-1.65,38.21,18.67,36.06,34.17Z" transform="translate(207.26 95.83)"/><path class="cls-2" d="M108.09,65.74c-10.26-8.17-20.44-8.9-30.62.41L70.56,53.66C83.13,42.05,102.85,42,115,53.73Z" transform="translate(207.26 95.83)"/></svg>
```
Comment (squidbeaks) [inline assets/images/grape-logo.svg]: adding extra line to the end of this file (github yells at you when you don't)

File: assets/js/script.js [hunk 2]
```diff
@@ -1,20 +1,108 @@
...
+const randomPairingEl = document.querySelector('#randomPairing');
 
+// hardcoded examples
+let info = "I am a full bodied wine made with grapes. Has been fermenting since the days of old";
+let rating = '⭐️⭐️⭐️⭐️⭐️';
 
+//create a function to remove all html elements except the footer
+const removeExistingElems = () => {
+  headerEl.innerHTML = '';
+	mainEl.innerHTML = '';
```
Comment (squidbeaks) [inline assets/js/script.js:18]: fixing indentation

File: assets/js/script.js [hunk 3]
```diff
@@ -1,20 +1,108 @@
...
-	var collapseElems = document.querySelectorAll('.collapsible');
-    var collapseInstances = M.Collapsible.init(collapseElems, options);
-});
+const listenerHandler = el => {
+	el.addEventListener('change', function() {
+		var title = this.options[this.selectedIndex].text;
+		// fetch function dependent on `el`
+		removeExistingElems();
+		// these will go inside the fetch function because that's where we'll receive input for ratings, descr, and recipes
+		infoCardGenerator(title,info, rating);
```
Comment (squidbeaks) [inline assets/js/script.js]: fixing space

File: assets/js/script.js [hunk 4]
```diff
@@ -1,20 +1,108 @@
...
+		// these will go inside the fetch function because that's where we'll receive input for ratings, descr, and recipes
+		infoCardGenerator(title,info, rating); 
+		recipeCardGenerator();
+	});
+};
+
+// refactoring information display function
+const infoCardGenerator = function(title, info, rating) {
+	const infoCardDiv = document.createElement('div');
+	infoCardDiv.setAttribute("id", "content");
```
Comment (squidbeaks) [inline assets/js/script.js:35]: An alternative way to do this is:

`infoCardDiv.id = 'content';`

File: assets/js/script.js [hunk 5]
```diff
@@ -1,20 +1,108 @@
...
+	$('.collapsible').collapsible();
 });
-        
\ No newline at end of file
+};
+
+const recipeCardGenerator = (recipe1, recipe2, recipe3) => {
+	const recipeCardDiv = document.createElement("div");
+	const recipeUl = document.createElement("ul");
+	recipeUl.setAttribute("class", "collapsible");
```
Comment (squidbeaks) [inline assets/js/script.js:78]: An alternative way to write this is

`recipeUl.className = 'collapsible';`

File: assets/js/script.js [hunk 6]
```diff
@@ -1,20 +1,108 @@
...
+	recipeCardDiv.classList.add("col", "s12", "m7");
```
Comment (squidbeaks) [inline assets/js/script.js:79]: An alternative way to write this is

`recipeCardDiv.classList = 'col s12 m7';`

File: assets/js/script.js [hunk 7]
```diff
@@ -1,20 +1,108 @@
...
+	}
+	$("#contentRow").append(recipeCardDiv);
+};
+
+
+//create something to display the 3 options to the right
+
+listenerHandler(wineSelectorEl);
+listenerHandler(foodSelectorEl);
+listenerHandler(randomPairingEl);
```
Comment (squidbeaks) [inline assets/js/script.js]: adding extra line at the end so github stops squawking

**Review by squidbeaks — APPROVED** [review]

**Review by Shygois — APPROVED** [review]

**Pull request merged**